### DISCUSSION TITLE
Improve UX of reaching maximum bonus spins on spinner

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#16715] [Q&A]
I have noticed that on some maps, people who spin the spinner at 477 rpm stop getting bonus points for the spinner closer to the end of the spinner. I found a correlation with the number of points a auto can get. The player cannot get more than this value. This is especially clearly seen on the Kuba oms.

Example 1 (Kuba oms) - https://youtu.be/lzCaFm33PmQ (replay - https://osu.ppy.sh/scores/osu/4046994126)
Example 2 (Enemy) - https://youtu.be/w0F__ynRpdA (replay - https://osu.ppy.sh/scores/osu/2427027068)

Upd. On the Kuba the spinner becomes much louder closer to the end.

**Comment by smoogipoo** [comment]
This is intentional, the number of bonus points is capped to 477 but lazer allows spinning faster than 477. The player's actually spinning at more like 800 RPM in lazer.

**Reply by Moltanicaa** [reply]
May in this case show the value "maximum!" and show the actual speed of rotation?
